site/layouts/partials/footer.html

```diff
@@ -1,56 +1,113 @@
-<footer class="bd-footer py-3 py-md-4 mt-5 bg-dark">
-  <div class="container-xxl py-1">
-    <div class="row">
-      <div class="col-lg-3 mb-3">
-        <p class="mb-1">This documentation is an adaptation made by Orange.</p>
-        <p class="mb-1">Original version designed and built with all the love in the world by the <a href="https://github.com/orgs/twbs/people">Bootstrap core team</a> with the help of <a href="https://github.com/twbs/bootstrap/graphs/contributors">their contributors</a>.</p>
-        <p class="mb-md-1">
-          Orange modified code licensed <a href="{{ .Site.Params.repo }}/blob/main/LICENSE" target="_blank" rel="license noopener">MIT</a>
-          —&nbsp;just like <a href="https://github.com/twbs/bootstrap/blob/main/LICENSE" target="_blank" rel="license noopener">Bootstrap</a>,
-          docs <a href="https://creativecommons.org/licenses/by/3.0/" target="_blank" rel="license noopener">CC BY 3.0</a>.
-        </p>
-        <ul class="bd-footer-links ps-0 mb-0">
-          <li class="list-inline-item me-3 mb-3 mb-md-0 d-block d-md-inline-block"><span>Currently v{{ .Site.Params.current_version }}</span></li>
-          <li class="list-inline-item mt-3 mt-md-0"><a href="https://www.netlify.com">
-            <img src="https://www.netlify.com/img/global/badges/netlify-dark.svg" alt="Deploys by Netlify" width="89" height="40"/>
-          </a></li>
-        </ul>
+<footer class="footer bg-dark navbar-dark">
+  <h2 class="visually-hidden">Boosted sitemap & information</h2>
+  <div class="container-xxl footer-nav">
+    <nav class="accordion accordion-dark" id="sitemapAccordion" aria-label="Boosted sitemap footer">
+      <div class="row">
+        <div class="footer-column col-md-2 offset-md-1">
+          <h3 class="accordion-header footer-heading" id="headingLinks">
+            <button class="accordion-button collapsed container-xxl px-1 d-md-none" type="button"
+                    data-bs-toggle="collapse" data-bs-target="#collapseLinks" aria-expanded="true"
+                    aria-controls="collapseLinks">
+              Links
+            </button>
+            <span class="d-none d-md-flex">Links</span>
+          </h3>
+          <div id="collapseLinks" class="container-xxl accordion-collapse collapse" data-bs-parent="#sitemapAccordion">
+            <ul class="navbar-nav">
+              <li><a class="nav-link" href="/" aria-describedby="headingLinks">Home</a></li>
+              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/"
+                     aria-describedby="headingLinks">Docs</a></li>
+              <li><a class="nav-link" href="/docs/{{ .Site.Params.docs_version }}/examples/"
+                     aria-describedby="headingLinks">Examples</a></li>
+              <!-- Boosted mod: removed .Site.Params.themes and .Site.Params.blog -->
+            </ul>
+          </div>
+        </div>
+        <div class="footer-column col-md-2">
```

Comment:
We should maybe consider using `col-md-6 col-lg-2` since some links are not on one line -> Need to change the last paragraphs management